diff -aNur bundle.orig/headers-more-nginx-module-0.33/src/ngx_http_headers_more_headers_in.c bundle/headers-more-nginx-module-0.33/src/ngx_http_headers_more_headers_in.c
--- bundle.orig/headers-more-nginx-module-0.33/src/ngx_http_headers_more_headers_in.c	2017-11-04 07:05:42.000000000 +0800
+++ bundle/headers-more-nginx-module-0.33/src/ngx_http_headers_more_headers_in.c	2023-07-20 21:06:47.063956624 +0800
@@ -159,7 +159,7 @@
 #endif
 
     { ngx_string("Cookie"),
-                 offsetof(ngx_http_headers_in_t, cookies),
+                 offsetof(ngx_http_headers_in_t, cookie),
                  ngx_http_set_builtin_multi_header },
 
     { ngx_null_string, 0, ngx_http_set_header }
diff -aNur bundle.orig/nginx-1.21.4/src/event/ngx_event_udp.h bundle/nginx-1.21.4/src/event/ngx_event_udp.h
--- bundle.orig/nginx-1.21.4/src/event/ngx_event_udp.h	2023-07-21 14:11:07.949886373 +0800
+++ bundle/nginx-1.21.4/src/event/ngx_event_udp.h	2023-07-20 21:13:53.609658889 +0800
@@ -43,7 +43,7 @@
 #endif
 
 #if (NGX_HAVE_INET6 && NGX_HAVE_IPV6_RECVPKTINFO)
-    struct in6_pktinfo    pkt6;
+    //struct in6_pktinfo    pkt6;
 #endif
 } ngx_addrinfo_t;
 
diff -aNur bundle.orig/ngx_lua-0.10.21/src/ngx_http_lua_headers_in.c bundle/ngx_lua-0.10.21/src/ngx_http_lua_headers_in.c
--- bundle.orig/ngx_lua-0.10.21/src/ngx_http_lua_headers_in.c	2022-03-02 14:54:22.000000000 +0800
+++ bundle/ngx_lua-0.10.21/src/ngx_http_lua_headers_in.c	2023-07-20 21:04:49.834304821 +0800
@@ -153,7 +153,7 @@
 #endif
 
     { ngx_string("Cookie"),
-                 offsetof(ngx_http_headers_in_t, cookies),
+                 offsetof(ngx_http_headers_in_t, cookie),
                  ngx_http_set_builtin_multi_header },
 
     { ngx_null_string, 0, ngx_http_set_header }
